fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/MixinMobEntity.java

```diff
@@ -0,0 +1,56 @@
+package net.fabricmc.fabric.mixin.content.registry;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.mob.MobEntity;
+import net.minecraft.entity.player.ItemCooldownManager;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+
+import net.fabricmc.fabric.api.registry.ShieldRegistry;
+
+@Mixin(MobEntity.class)
+public abstract class MixinMobEntity {
+	/**
+	 * Also disable modded shields.
+	 */
+	@Redirect(method = "disablePlayerShield", at = @At(value = "INVOKE", target = "Lnet/minecraft/item/ItemStack;getItem()Lnet/minecraft/item/Item;"))
+	private Item disableFabricShields(ItemStack itemStack) {
+		if (itemStack.getItem() == Items.SHIELD || ShieldRegistry.INSTANCE.isShield(itemStack.getItem())) {
+			return Items.SHIELD;
```

Comment:
My assumption here is that you return shield to fulfill the `item == Items.SHIELD` call in `disablePlayerShield` method. I'd probably document that logic in this mixin